content/sensu-go/6.1/observability-pipeline/observe-schedule/subscriptions.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Subscriptions reference"
+linkTitle: "Subscriptions Reference"
+reference_title: "Subscriptions"
+type: "reference"
+description: "Sensu subscriptions are equivalent to topics in a traditional publish/subscribe system. With Sensu subscriptions, you can configure check requests in a one-to-many model for an entire group or subgroup of systems rather than a traditional one-to-one mapping of configured hosts or observability checks. Read this reference doc to learn how to configure subscriptions in Sensu."
+weight: 35
+version: "6.1"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-6.1:
+    parent: observe-schedule
+---
+
+Sensu uses the [publish/subscribe model of communication][1].
+The publish/subscribe model is powerful in ephemeral or elastic infrastructures, where the names and numbers of things change over time.
+
+Because Sensu uses the publish/subscribe model, you can write checks even if you don't know the specific names of the entities that should run the checks.
+Likewise, your entities do not need to know the specific names of the checks they should execute.
+The Sensu backend coordinates check execution for you by comparing the subscriptions you specify in your checks and entities to determine which entities should receive execution requests for a given check.
+
+The diagram below shows how Sensu coordinates check execution based on subscriptions.
+For example, the `check_cpu` check includes the `system` subscription.
+All three entities include the `system` subscription, so all three entities will execute the `check_cpu` check.
+However, only Entity A and Entity C will execute `checkSshdProcess` &mdash; Entity B does not include the `linux` subscription required to execute `checkSshdProcess`.
+
+{{< figure src="/images/subscriptions_venn.png" alt="Venn diagram example of Sensu check execution based on subscriptions" link="/images/subscriptions_venn.png" target="_blank" >}}
+<!-- Diagram source: https://app.lucidchart.com/invitations/accept/0aa33f39-128d-449d-8312-04f415d8cfca -->
```

Comment:
```suggestion
<!-- Diagram source: https://app.lucidchart.com/documents/view/4ed31993-de03-46ed-92b9-0d9eed176439 -->
```

I think we should be using links to `documents` in LucidChart. The links with `invitations/accept` might grant access to documents which we don't want to provide to just anybody.

Then again, I can't tell from the link whether the invitation is simply to view the document, which would be a non-issue.

Reply:
I don't know why or when this changed, but I can't seem to generate a LucidChart link that doesn't include "invitations/accept" anymore. The links I'm generating are all restricted to Sensu though.